### DISCUSSION TITLE
Android apk

### DISCUSSION THREAD

**Discussion by LAS-Soft** [vvip-68/GyverPanelWiFi#310] [Задавайте вопросы]
хотел скачать актуальный GyverPlayerWiFi . арк но система выдала "осторожно вирус" ранее вроде ничего подобного не выдавало. Актуальная версия приложения не перезалита сторонними лицами?

**Comment by vvip-68** [comment]
Да никто не имеет доступа на запись в репозиторий кроме автора.
Хотите - возьмите aia файл, загрузите в appinventor, скомпилте apk там
Видимо у вас антивирусник с паранойей...

НЕ понимаю, чего все к версии 1.13 привязались?
Вроде 1.14 поинтереснее, приложений не нужно - в браузере управление...

**Reply by LAS-Soft** [reply]
речь про https://github.com/vvip-68/LedPanelWiFi/tree/main ? на нее несколько сложнее выйти чем на 1.13
сейчас скачаю, посмотрю, попробую

**Reply by shkolin1973** [reply]
Привязались потому, что со смартфона управлять гирляндой удобнее через приложение, а не через web  интерфейс. Наверно поэтому для смартфонов есть приложения для всех сервисов и ресурсов. На компе мы везде заходим через браузер, а на смарте через приложения. Хотя Web интерфейс на 1,14 весьма хорош, но приложение на 1,13 (лично для меня) удобнее. Только по этой причине я на версии 1,13.

**Comment by vvip-68** [comment]
Мне кажется на самой первой страничее репозитория 1.13 в начале написано, что проект устарел и не поддерживается и дана ссылка на 1.14... Разве нет?

**Reply by LAS-Soft** [reply]
захожу по ссылке в избранном и верхнюю часть как-то не читал:(
Хром обрывает закачку как это ни странно
![image](https://github.com/user-attachments/assets/c42a1049-6af7-4dc5-9109-daae794c2d6a)

**Reply by vvip-68** [reply]
Ну у вас у антивируса паранойя. Какой вирус в текстовых файлах?

**Reply by LAS-Soft** [reply]
скачал, залил. сперва вроде все Ок (хочу сделать просто длинную гирлянду) а потом произошло какое-то залипание из которого смог извлечь только прошив на 1.13
насколько понял по монитору порта он пытался запустить погоду, и перезапускал устройство каждый раз

**Reply by vvip-68** [reply]
Ничё не понятно

**Reply by LAS-Soft** [reply]
почему то гоняет по кругу:
14:00:08.353 -> --------------- CUT HERE FOR EXCEPTION DECODER ---------------
14:00:08.353 -> 
14:00:08.353 ->  ets Jan  8 2013,rst cause:2, boot mode:(3,7)
14:00:08.353 -> 
14:00:08.353 -> load 0x4010f000, len 3584, room 16 
14:00:08.353 -> tail 0
14:00:08.353 -> chksum 0xb0
14:00:08.353 -> csum 0xb0
14:00:08.353 -> v2843a5ac
14:00:08.353 -> ~ld
14:00:08.831 -> 
14:00:08.831 -> WiFiPanel v.1.13.2023.1111
14:00:08.831 -> Версия EEPROM: 0x5D
14:00:08.831 -> Host: 'WiFiPanel'
14:00:08.831 -> 
14:00:08.831 -> Инициализация файловой системы... 
14:00:08.831 -> Использовано 16384 из 2072576 байт
14:00:08.831 -> 
14:00:08.831 -> Загрузка строк выполнена.
14:00:08.831 -> Использованы адреса EEPROM 1000 - 1037
14:00:08.831 -> Свободно ячеек 3058
14:00:08.831 -> 
14:00:08.831 -> Матрица: плоская 112x1
14:00:08.831 -> Адресация: по подключению
14:00:08.831 ->   Угол: левый нижний
14:00:08.831 ->   Направление: вправо
14:00:08.831 ->   Тип: зигзаг
14:00:08.831 -> 
14:00:08.831 -> Доступные возможности:
14:00:08.831 -> + Бегущая строка: шрифт 5x8
14:00:08.831 -> + Кнопка управления: тактовая
14:00:08.831 -> + Синхронизация времени с сервером NTP
14:00:08.831 -> - Управление питанием
14:00:08.831 -> + Получение информации о погоде
14:00:08.831 -> - Управление по каналу MQTT
14:00:08.831 -> + Групповая синхронизация по протоколу E1.31
14:00:08.831 -> - Дополнительный индикатор TM1637
14:00:08.831 -> - Эффекты Jinx! с SD-карты
14:00:08.831 -> - Поддержка MP3 Player
14:00:08.831 -> 
14:00:08.831 -> Поиск файлов карт индексов...
14:00:08.831 -> Нет файлов карт индексов
14:00:08.831 -> 
14:00:08.831 -> sMATRIX_TYPE=0
14:00:08.831 -> 
14:00:08.831 -> Найденные анимации:
14:00:08.831 ->    Сердце	8x8
14:00:08.831 -> 
14:00:08.831 -> Поиск слайдов на FS в папке '112p1'...
14:00:08.831 -> Поиск слайдов на FS в папке '16p16'...
14:00:08.831 -> Слайды не найдены. Будут использованы картинки эффекта 'Погода'
14:00:08.831 -> 
14:00:09.253 -> Создание точки доступа PanelAP
14:00:11.454 -> Точка доступа создана. Сеть: 'PanelAP'. Пароль: '12341234'.
14:00:11.454 -> IP адрес: 192.168.4.1
14:00:11.454 -> 
14:00:11.454 -> UDP-сервер на порту 2390
14:00:11.454 -> 
14:00:11.454 -> Групповая синхронизация по протоколу E1.31
14:00:11.454 -> Режим работы: АВТОНОМНЫЙ, синхронизация E1.31 отключена
14:00:11.454 -> 
14:00:11.454 -> Переход в авторежим отключен
14:00:11.454 -> Режим: Палитра
14:00:11.454 -> 
14:00:11.454 -> --------------- CUT HERE FOR EXCEPTION DECODER ---------------

**Reply by vvip-68** [reply]
Вот что-то даже разбираться не буду что там в старой версии случается. 
Ставьте Exception Decoder, копируйте туда ошибку, пытайтесь понять что идет не так...
Или просто поставьте последнюю версию, возможно там это исправлено или может просто не приводит к такой ошибке.
А ковыряться с версией годовалой давности - не... Не буду...

**Reply by LAS-Soft** [reply]
я заливал 1.14 и она начала бегать примерно по такому кругу:
 -> Найденные анимации
 -> Поиск слайдов на FS в папке '112p1'...
 -> Поиск слайдов на FS в папке '16p16'...
 -> Слайды не найдены. Будут использованы картинки эффекта 'Погода'
 -> Создание точки доступа PanelAP
 -> Точка доступа создана. Сеть: 'PanelAP'. Пароль: '12341234'.
 -> IP адрес: 192.168.4.1

Я пробовал отключать питание, перезаливать 1.14, но ардуинка просто бегала по кругу, и только почему-то на 1.13 перешилась, так что пока оставляю старую, а для 1.14 закажу пару новых платок

**Reply by vvip-68** [reply]
Да у вас даже 1.13 годовалой давности:
```
14:00:08.831 -> WiFiPanel v.1.13.2023.1111
14:00:08.831 -> Версия EEPROM: 0x5D
```

Не понял что не так с 1.14 - вроде нормальный вывод в лог.

**Reply by LAS-Soft** [reply]
попробую описать словами:
прошивка залилась, гирлянда заработала, я решил проверить переключение режимов с кнопки, переключалось норм, при очередном нажатии гирлянда погасла. Сперва подумал что просто выключил, но обратно включит не удавалось и тут я заметил что раз в несколько секунд вспыхивал синий диод на плате. А т.к. она еще была подключена к компу то в мониторе я увидел что она не находит анимацию, и запускает создание точки доступа после чего снова ищет анимацию, не находит и опять начинает создавать точку доступа. и так бесконечно

**Reply by vvip-68** [reply]
Не подскажу - не сталкивался с таким...
